dbprocessing/Utils.py

```diff
@@ -560,3 +560,18 @@ def readconfig(config_filepath):
                 else:
                     ans[section][item] = (ans[section][item], 0, 0)
     return ans
+def load_source(modname, filepath, module):
```

Comment:
PEP8 specifies space (i.e. one empty line) between the functions.

Can you document parameters/returns as in the other functions?

The module argument doesn't appear to be used--can it be removed? (And then update calls to it, of course).